specification/automation/resource-manager/Microsoft.Automation/stable/2022-01-31/deletedAutomationAccount.json

```diff
@@ -0,0 +1,147 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "AutomationManagement",
+    "version": "2022-01-31",
+    "x-ms-code-generation-settings": {
+      "useDateTimeOffset": true
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts/{automationAccountName}": {
+      "get": {
+        "tags": [
+          "DeletedAutomationAccounts"
+        ],
+        "operationId": "deletedAutomationAccounts_ListBySubscription",
+        "description": "Retrieve deleted automation account.",
+        "x-ms-examples": {
+          "Get deleted automation account": {
+            "$ref": "./examples/getDeletedAutomationAccount.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DeletedAutomationAccountListResult"
```

Comment:
The URI for this operation should be returning a single deleted account with name PautomationAccountName}.  Not a list.  LIST URIs do not include the name of the resource

Reply:
URI return list of deleted account with in a subscription. 
deletedAutomationAccounts is a resource type in /subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts URI.

Reply:
The path for this GET includes the name of a single account.  You changed it in this PR from what it used to be (which was correctly a LIST call).  It is now "/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts/{automationAccountName}".  It should probably be reverted and a new path added for the individual GET

Reply:
/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts API returns list of deleted automation accounts with in a subscription. deletedAutomationAccounts is the new resource type that was added. Path doesn't contain single account name. GET returns list of previously deleted account which customer can use to restore deleted account.

Reply:
The URI path in this PR is `/subscriptions/{subscriptionId}/providers/Microsoft.Automation/deletedAutomationAccounts/{automationAccountName}`.  This path should return 1 account.  If you want to return a list of accounts then remove `{automationAccountName}` from the URI

Reply:
Fixed the URI, Thanks Chris